### PR TITLE
Update/dataproc persistent logging

### PR DESCRIPTION
**Pull request by jaketf**

### CODE REVIEW

**Comment by jaketf** [comment]
@yhqs540 can you please review these updates when you get a chance?

File: examples/dataproc-persistent-history-server/init_actions/disable_history_servers.sh [hunk 1]
```diff
@@ -1,7 +1,13 @@
 #!/bin/bash
 set -euxo pipefail
 role="$(/usr/share/google/get_metadata_value attributes/dataproc-role)"
-if [[ "${role}" == 'Master' ]]; then
-  systemctl stop hadoop-mapreduce-historyserver
+
+if [[ "${role}" == "Master" && -f /etc/init.d/spark-history-server ]]; then
```
Comment (yhqs540) [inline examples/dataproc-persistent-history-server/init_actions/disable_history_servers.sh]: Small thing. Maybe extract the /etc/ entries to a separate variable.

File: examples/dataproc-persistent-history-server/init_actions/disable_history_servers.sh [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+
   systemctl stop spark-history-server
 fi
+
+if [[ "${role}" == "Master" && -f /etc/init.d/hadoop-mapreduce-historyserver ]]; then
```
Comment (yhqs540) [inline examples/dataproc-persistent-history-server/init_actions/disable_history_servers.sh]: same as above

**Comment by jaketf** [comment]
@yhqs540 variablized init paths: 3e07ef8

**Pull request merged**